tests/Behat/Context/Ui/Admin/ManagingChannelsSetupContext.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Tests\Sylius\InvoicingPlugin\Behat\Page\Admin\Channel\UpdatePageInterface;
+
+final class ManagingChannelsSetupContext implements Context
```

Comment:
IMO, you should add `ChannelContext` in `Tests\Sylius\InvoicingPlugin\Behat\Context\Setup\` and set this data on `Channel` entity. Look at [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Context/Setup/ChannelContext.php).